wapi/session.py

```diff
@@ -134,7 +134,7 @@ def get_curve(self, id=None, name=None):
 
     def search(self, query=None, id=None, name=None, commodity=None, category=None, area=None, station=None,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Method "search" has 18 parameters, which is greater than the 15 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.dev.wattsight.com/coding_rules#rule_key=python%3AS107)